### PR TITLE
chore: Add env setup instructions to README

### PR DESCRIPTION
**Pull request by david-code**
## Description

Adds info for setting up mobile env

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,3 +111,18 @@ Sign the app bundle:
...
+
+# Env Setup
+
+## Logging In
+
+Once all of the dependencies are installed, the first barrier to running the project locally is getting past the login screen. In order to do so, you'll need to do the following:
+
+1. Set up a Google OAuth project for Android or iOS
+- See [Google Cloud Console](https://console.cloud.google.com/)
+2. Copy `.env.sample` to `.env`. You need to change the `GOOGLE_OAUTH_APP_CLIENT_ID` variable to match
```
Comment (paulschreiber) [inline README.md:123]: Please clarify `change to match`.

File: README.md [hunk 2]
```diff
@@ -111,3 +111,18 @@ Sign the app bundle:
 ```
 jarsigner -verbose -sigalg SHA256withRSA -digestalg SHA-256  -keystore ~/terraso-lpks-key.keystore -signedjar app/build/outputs/bundle/release/terraso-landpks.aab  app/build/outputs/bundle/release/app-release.aab terraso-lpks
 ```
+
+# Env Setup
+
+## Logging In
+
+Once all of the dependencies are installed, the first barrier to running the project locally is getting past the login screen. In order to do so, you'll need to do the following:
```
Comment (paulschreiber) [inline README.md:119]:
```suggestion
Once all of the dependencies are installed, you'll need to do the following to enable logins:
```

**Review by paulschreiber — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -111,3 +111,18 @@ Sign the app bundle:
...
+
+# Env Setup
+
+## Logging In
+
+Once all of the dependencies are installed, you'll need to do the following to enable logins:
+
+1. Set up a Google OAuth project for Android or iOS if it doesn't already exist
+- See [Google Cloud Console](https://console.cloud.google.com/)
+2. Copy `.env.sample` to `.env`. You need to change the `GOOGLE_OAUTH_APP_CLIENT_ID` variable to match the value of your OAuth App client id
```
Comment (paulschreiber) [inline README.md:123]:
```suggestion
2. Copy `.env.sample` to `.env`. Change `GOOGLE_OAUTH_APP_CLIENT_ID` variable to match the value of your OAuth App client ID in Google Cloud Console.
```

**Review by paulschreiber — APPROVED** [review]

**Review by shrouxm — APPROVED** [review]
LGTM, i will give it a whirl when i'm doing my next home screen PR!

**Pull request merged**